homeassistant/helpers/config_validation.py

```diff
@@ -1804,7 +1828,9 @@ def _base_trigger_validator(value: Any) -> Any:
 
 
 TRIGGER_SCHEMA = vol.All(
-    ensure_list, _base_trigger_list_flatten, [_base_trigger_validator]
+    ensure_list,
+    _base_trigger_list_flatten,
+    [vol.All(_base_trigger_validator, _backward_compat_trigger_schema)],
```

Comment:
Since `trigger` is now yaml only, can we run the `_backward_compat_trigger_schema` before we run `_base_trigger_validator`?